e2e-testing/cypress/functions/LoginFunctions.js

```diff
@@ -25,6 +25,15 @@ class LoginFunctions {
     cy.get(LoginLocators.loginButton).click();
   }
 
+  check_user_details_correct(){
```

Comment:
I don't see where this method is used. @shakirandagire 
Was this implementation complete?

Reply:
I had completed the implementation and put the method in the cypress file. Let me check and see what happenned